test/test_verbs.rb

```diff
@@ -398,4 +398,9 @@ def test_passive
       Verbs::Conjugator.conjugate(:be, diathesis: :passive)
     end
   end
+
+  def test_not_modifying_frozen_string
+    assert_equal 'is diying', Verbs::Conjugator.conjugate(:die, aspect: :progressive)
```

Comment:
@nerixim can you please comment on this? `diying` is not correct unless I'm misunderstanding the test.

Reply:
Oops I forgot you couldn't conjugate `die` yet.
I'll change these examples with another verb.